config/cypress.json

```diff
@@ -10,6 +10,7 @@
   "testFiles": "**/tests/**/*.cypress.spec.js?(x)",
   "fixturesFolder": "src",
   "waitForAnimations": false,
+  "chromeWebSecurity": false,
```

Comment:
Is there a way to disable this check only for certain tests? It seems like it could still be valuable to flag bad behavior in other apps

Reply:
That makes sense, thanks for the explanation!